comp-delegations-monitor/.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+forta.config.json
```

Comment:
```suggestion
forta.config.json

```
minor nit.

Reply:
I did not know that, thanks for explaining!